setup.py

```diff
@@ -20,6 +20,7 @@
         'cryptography>=1.7.2',
         'pyOpenSSL>=16.2.0',
         'msgpack-python>=0.4.2',
+        'filemagic'
```

Comment:
Hmmm... didn't take that into account. It looks like there is a libmagic port to windows, which may be sufficient. I'll look into this a bit more.